lib/Workflow.groovy

```diff
@@ -15,12 +15,42 @@ class Workflow {
                "  https://github.com/${workflow.manifest.name}/blob/master/CITATIONS.md"
     }
 
+    // Print help to screen if required
+    public static String help(workflow, params, json_schema, log) {
+        def command = "nextflow run nf-core/rnaseq --input samplesheet.csv --genome GRCh37 -profile docker"
+        def help_string = ''
+        help_string += Utils.logo(workflow, params.monochrome_logs)
+        help_string += NfcoreSchema.paramsHelp(workflow, params, json_schema, command)
+        help_string += '\n' + Workflow.citation(workflow) + '\n'
+        help_string += Utils.dashedLine(params.monochrome_logs)
+        return help_string
+    }
+
+    // Print parameter summary log to screen
+    public static String paramsSummaryLog(workflow, params, json_schema, log) {
+        def summary_log = ''
+        summary_log += Utils.logo(workflow, params.monochrome_logs)
+        summary_log += NfcoreSchema.paramsSummaryLog(workflow, params, json_schema)
+        summary_log += '\n' + Workflow.citation(workflow) + '\n'
+        summary_log += Utils.dashedLine(params.monochrome_logs)
+        return summary_log
+    }
+
     public static void validateMainParams(workflow, params, json_schema, log) {
```

Comment:
I'd be tempted to call this something else, as it does a lot more than just validate params now. `initialiseWorkflow` or `startYourEngines` or something? 😅

Reply:
Done https://github.com/drpatelh/rnaseq/blob/f1f5e891d4bcfd9c45965857b33b3cc66f2c2ce6/main.nf#L31

Reply:
Now `WorkflowMain.initialise(workflow, params, log)`